.github/workflows/update-drci-comments.yml

```diff
@@ -0,0 +1,22 @@
+name: Update Dr. CI comments
+
+on:
+  schedule:
+    # Run every 15 minutes
+    - cron: "*/15 * * * *"
+  # Have the ability to trigger this job manually through the API (temporary)
+  workflow_dispatch:  
+
+jobs:
+  update-drci-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Retrieve rockset query results and update Dr. CI comments
+        run: |
+          curl --request POST \
+          --url 'https://www.torch-ci.com/api/drci/drci' \
```

Comment:
nit: if we're going with the api, I don't think we need to repeat drci twice. If you want to put it in it's own directory, you might be able to name the file index.ts or something like that to not have the drci/drci thing.